lib/ui/page/home/page/user/view.dart

```diff
@@ -244,7 +244,7 @@ class UserView extends StatelessWidget {
 
                 return Padding(
                   padding: const EdgeInsets.fromLTRB(8, 0, 8, 4),
-                  child: _blockedField(context, c),
+                  child: UserBlockedField(unblacklist: () => c.unblacklist()),
```

Comment:
```suggestion
                  child: UserBlockedField(unblacklist: c.unblacklist),
```

Зачем анонимную функцию создавать, если можно дать ссылку на имеющуюся?

Reply:
Вы правы.